internal/cmd/server.go

```diff
@@ -240,3 +259,12 @@ func setupRunners(ctx context.Context, cfg *config.Server) (err error) {
 
 	return nil
 }
+
+func startRunner(ctx context.Context, cfg *config.Server, runner Runner) error {
+	cfg.Logger.Info("Starting runner", "name", runner.Name())
```

Comment:
can we avoid this line, feel redundant since we have these inside the runner itself
https://github.com/envoyproxy/gateway/blob/1ef27f7c2764a18c4bf0f7330840c30397010de9/internal/gatewayapi/runner/runner.go#L82